packages/data-store/src/index.ts

```diff
@@ -66,6 +67,7 @@ export const DataStoreContactEntities = [
   NaturalPersonEntity,
   ElectronicAddressEntity,
   PhysicalAddressEntity,
+  ContactMetadataItemEntity,
```

Comment:
needs to be exported further down as well

Reply:
Fixed